### PR TITLE
Splashtop.SplashtopPersonal version 3.46.9.7601

### PR DESCRIPTION
**Pull request by circlol**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37020)

### CODE REVIEW

File: manifests/s/Splashtop/SplastopPersonal/3.46.9.7601/Splashtop.SplastopPersonal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Splashtop.SplastopPersonal
```
Comment (dorssel) [inline manifests/s/Splashtop/SplastopPersonal/3.46.9.7601/Splashtop.SplastopPersonal.installer.yaml:4]: That looks like a spelling mistake in the package name... (missing 'h'?)

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/37020#discussion_r762811606

File: manifests/s/Splashtop/SplastopPersonal/3.46.9.7601/Splashtop.SplastopPersonal.installer.yaml (hunk 1 above)
Comment (circlol) [reply manifests/s/Splashtop/SplastopPersonal/3.46.9.7601/Splashtop.SplastopPersonal.installer.yaml]: fixed

**Comment by OfficialEsco** [comment]
```
##[error] Result: Failed
##[error] Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual   'manifests\s\Splashtop\SplastopPersonal\3.46.9.7601', 
expected 'manifests\s\Splashtop\SplashtopPersonal\3.46.9.7601'
##[error]Manifest Schema Validation Failed
```

**Pull request closed without merging**